vector/src/main/java/im/vector/app/features/widgets/webview/WebviewPermissionUtils.kt

```diff
@@ -15,17 +15,33 @@
  */
 package im.vector.app.features.widgets.webview
 
+import android.Manifest
 import android.annotation.SuppressLint
 import android.content.Context
 import android.webkit.PermissionRequest
+import androidx.activity.result.ActivityResultLauncher
 import androidx.annotation.StringRes
+import androidx.annotation.VisibleForTesting
+import androidx.fragment.app.FragmentActivity
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import im.vector.app.R
+import im.vector.app.core.utils.checkPermissions
+import java.lang.NullPointerException
+import javax.inject.Inject
 
-object WebviewPermissionUtils {
+class WebviewPermissionUtils @Inject constructor() {
+
+    private var permissionRequest: PermissionRequest? = null
+    private var selectedPermissions = listOf<String>()
 
     @SuppressLint("NewApi")
```

Comment:
I am not sure we still need this annotation. I have run the lint task and I had no error...

Reply:
Removed it